include/devices/wifi_device.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Race Capture Firmware
+ *
+ * Copyright (C) 2016 Autosport Labs
+ *
+ * This file is part of the Race Capture firmware suite
+ *
+ * This is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details. You should
+ * have received a copy of the GNU General Public License along with
+ * this code. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef WIFI_DEVICE_H_
+#define WIFI_DEVICE_H_
+
+#include "cpp_guard.h"
+#include <stdbool.h>
+
+CPP_GUARD_BEGIN
+
+bool wifi_device_init();
+void wifi_device_reset();
```

Comment:
well, since it's at the HAL layer it's pretty much the only thing that could happen. Thoughts?